tests/ai_safety/evalhub/utils.py

```
@@ -594,6 +594,31 @@ def delete_evalhub_job(
+    records never fail a test or leak into subsequent tests.
+    """
+    try:
+        delete_evalhub_job(
+            host=host,
+            token=token,
+            ca_bundle_file=ca_bundle_file,
+            tenant=tenant,
+            job_id=job_id,
+            hard_delete=True,
+        )
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)